client/src/components/Views/DayViewContent.js

```diff
@@ -0,0 +1,72 @@
+import { useCallback, useEffect, useMemo, useState } from 'react'
+import TripViewJson from '../../temp/tripViewData.json'
+import { useDispatch, useSelector } from 'react-redux'
+import { Button } from '../common'
+import { setAppView } from '../../redux/reducers/viewSlice'
+import { AppView } from '../../constants/enums'
+
+export function DayViewContent() {
+  const activeDayNumber = useSelector((state) => state.view.activeDayNumber)
+  const dispatch = useDispatch()
+
+  const [stages, setStages] = useState([])
+
+  useEffect(() => {
+    // TODO edit this entire block to fetch from backend
+    const mockData = TripViewJson[activeDayNumber - 1]
+    setStages(mockData['stages'])
+  }, [activeDayNumber])
+
+  const generateStage = useCallback((stageInfo, index) => {
+    return (
+      <div className='grid grid-cols-5 py-2' key={`stage-${index}`}>
+        <div className='z-[5] w-20 text-center text-5xl'>
```

Comment:
We could use mx-auto later on to have the emoji responsive, so adjusting screen size won't hide location name and description

Reply:
good idea looking into it

Reply:
Couldnt get container working but added a screen-responsive breakpoint for now

![image](https://github.com/cpsc455-bugstorm/TravelersTea/assets/30674558/34e3eca6-3c3b-46ab-bb01-c6a5dc78ec62)